task_DEV_1/task_DEV_1/Program.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System;
+
+namespace task_DEV_1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter A Number :");
+            int index = int.Parse(Console.ReadLine());
+            if (index % 2 != 0 && index % 5 != 0 && (index % 2 != 0 || index % 5 != 0))
```

Comment:
Is it necessary to do this checks? Looks like it is overhead. Does program work incorrect without this?

Reply:
Instead of this check you can add just else block where you will output phrase that input number can't be divided on 2 or 5. Smth like this.